app/assets/stylesheets/modules/top_navbar.scss

```diff
@@ -38,8 +37,36 @@
  }
 }
 
+.navbar {
+  max-height: 120px;
+
+  .header-bar {
+    flex-wrap: wrap !important;
+  }
+
+  #topnav-container {
```

Comment:
Is this redundant styling (or unexecuted)? I'm seeing similar things on LN3 but without the qualifying parent classes

Reply:
Unexecuted. That id no longer exists. Removing.